fe/fe-core/src/main/java/org/apache/doris/clone/TabletScheduler.java

```diff
@@ -1053,7 +1054,23 @@ private void selectTabletsForBalance() {
 
         List<TabletSchedCtx> alternativeTablets = rebalancer.selectAlternativeTablets();
         for (TabletSchedCtx tabletCtx : alternativeTablets) {
-            addTablet(tabletCtx, false);
+            TabletMeta tabletMeta = invertedIndex.getTabletMeta(tabletCtx.getTabletId());
+            if (tabletMeta == null) {
+                continue;
+            }
+            Database db = Catalog.getCurrentCatalog().getDb(tabletMeta.getDbId());
+            OlapTable table = db == null ? null : (OlapTable) db.getTable(tabletMeta.getTableId());
+            if (table != null) {
+                table.readLock();
+                try {
+                    if (table.getState() == OlapTableState.NORMAL) {
+                        addTablet(tabletCtx, false);
+                    }
+                } finally {
+                    table.readUnlock();
+                }
+            }
```

Comment:
We have a logic here:

https://github.com/apache/incubator-doris/blob/a2e83e65d2e8aad604c35fe492f5bdd92650cbd7/fe/fe-core/src/main/java/org/apache/doris/clone/TabletScheduler.java#L513-L516

so I think it is not neccesary to check here.